integration/tests/operators/06-operator-deployment.spec.js

```diff
@@ -19,7 +19,7 @@ test("Deploys an Operator", async ({ page }) => {
   // Select operator to deploy
   await page.locator("input#search").fill("prometheus");
```

Comment:
Alternatively, we could just navigate to `/#/c/default/ns/default/operators?Search=prometheus&Provider=Red%20Hat`, however I didn't want to make more changes in the tests as I hope we move the operators to a plugin soon 😅

Reply:
I would rather go with adding an instruction before to click the checkbox "Red Hat" in the providers filter.
With that and the search term, maybe it is less error prone? I assume it is less likely to happen that they change the Provider than the description, though.

Reply:
Yep, fair point. However, it seems we also have multiple matchings with the provider itself:

![image](https://user-images.githubusercontent.com/11535726/185916852-6e63ef96-5fda-4d1f-a222-900c017017c3.png)

But, sure, let's give it a try, I'll push a commit soon. CI passed with the current fix, so clearly, this was the reason behind the CI failure.